### ISSUE TITLE
Silencing irrelevant warnings

### ISSUE THREAD

**Issue by kylebgorman** [CUNY-CL/yoyodyne#11]
[copied from CUNY-CL/abstractness/issues/101]

I see the following (mostly spurious) warnings on various runs:

* PossibleUserWarning about number of data loaders
* ~UserWarning: deprecation of `nn.functional.sigmoid`~
* UserWarning: dropout expects num_layers > 1
* ~UserWarning: trying to infer the `batch_size` from an ambiguous collection~
* PossibleUserWarning: `max_epochs` was not set.

Users tend to treat warnings as errors, so we may want to suppress these.
